modules/gloo/pkg/vsfactory/router.go

```diff
@@ -226,10 +243,6 @@ func addRedirect(route riov1.RouteSpec, gatewayRoute *gatewayv1.Route) {
 		ra.RedirectAction.PathRewriteSpecifier = &gloov1.RedirectAction_PathRedirect{
 			PathRedirect: route.Redirect.Path,
 		}
-	} else if route.Redirect.Prefix != "" {
```

Comment:
Prefix is just handled via match now ?

Reply:
prefix is handled by route.Redirect.Path